common/config.py

```diff
@@ -0,0 +1,7 @@
+import configparser
+
+
+class Configs:
```

Comment:
Again, up to your tastes but you can name the file configs and use it as a namespace. I understand using classes though.